### ISSUE TITLE
API errors

### ISSUE THREAD

**Issue by MoeFuGithub** [TheLastBen/fast-stable-diffusion#1910]
Calculating sha256 for /content/gdrive/MyDrive/Models/3moonNIAni_3moonNIAni.ckpt: 0d6ba968b6ebc1a47654cd0be7a359adeb174b59e283cad842a20a8d3ec3cec0
Loading weights [0d6ba968b6] from /content/gdrive/MyDrive/Models/3moonNIAni_3moonNIAni.ckpt
Creating model from config: /content/gdrive/MyDrive/sd/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Downloading (…)olve/main/vocab.json100% 961k/961k [00:00<00:00, 4.85MB/s]
Downloading (…)olve/main/merges.txt100% 525k/525k [00:00<00:00, 3.15MB/s]
Downloading (…)cial_tokens_map.json100% 389/389 [00:00<00:00, 114kB/s]
Downloading (…)okenizer_config.json100% 905/905 [00:00<00:00, 272kB/s]
Downloading (…)lve/main/config.json100% 4.52k/4.52k [00:00<00:00, 766kB/s]
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(0): 
Model loaded in 143.0s (calculate hash: 87.6s, load weights from disk: 46.3s, create model: 3.9s, apply weights to model: 3.5s, apply half(): 0.9s, move model to device: 0.7s).
Running on public URL: https://17edc2bcc36a4e4f83.gradio.live/
✔ Connected
Startup time: 180.3s (import torch: 14.3s, import gradio: 1.6s, import ldm: 0.7s, other imports: 6.4s, list SD models: 7.2s, setup codeformer: 0.4s, load scripts: 2.4s, load SD checkpoint: 143.0s, create ui: 1.7s, gradio launch: 2.3s, scripts app_started_callback: 0.1s).
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=script.js?1680193699.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/textualInversion.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/imageviewer.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/ui.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/aspectRatioOverlay.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/extensions.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/edit-attention.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/extraNetworks.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/generationParams.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/hires_fix.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/imageMaskFix.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/dragdrop.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/localization.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/progressbar.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/hints.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=extensions-builtin/prompt-bracket-checker/javascript/prompt-bracket-checker.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/contextMenus.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=style.css?1680193699.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/imageParams.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}
API error: GET: https://17edc2bcc36a4e4f83.gradio.live/file=javascript/notification.js?1680193698.0 {'error': 'HTTPException', 'detail': 'Not authenticated', 'body': '', 'errors': ''}

**Comment by TheLastBen** [comment]
did it work before ?

**Comment by biscuitderg** [comment]
got similar API errors when starting SD, haven't used the notebook before today. still works and runs txt2img generation, but inpaint/img2img doesn't seem to be working properly

**Comment by TheLastBen** [comment]
use ngrok and check the box "use_latest_working_commit"

**Comment by biscuitderg** [comment]
i used localtunnel, still got the api error on startup but the web ui is working as expected so far

**Comment by protrading104** [comment]
Additionally, there are issues with the API. I am trying to connect the stable.art plugin to Photoshop, which requires granting access through the API.
![api_error](https://user-images.githubusercontent.com/71656833/228937757-4802b071-9ee6-48c8-b32c-0441fd7fc3a0.png)

**Comment by Taikakim** [comment]
Yes, also since a few days ago, a ton of errors of all kind are appearing, I'm losing the connection to the notebook all the time, etc. Things have got very broken suddenly :/ It' s shame that on Colab I don't think there's a way to freeze the situation with the notebook anyhow?

![Uploading Untitled.png…]()
